components/tekton/tasks/deploy-gotests.yaml

```diff
@@ -14,28 +14,33 @@ metadata:
 spec:
   description: >-
     Beware! Task for demoing and testing purposes ONLY.
+    Suggestion: This task should be decomposed in simpler steps.
     Not intended for being a generic task to run any testsuite.
     This task will run gotest suite specified by FEATURE on a CLUSTER.
     Requires a kubeconfig of a cluster with two worker nodes with
-    SR-IOV and PTP capable NICs.
-    This task should be decomposed in simpler steps.
-    Available features supported now in this task is only SR-IOV .
+    identical SR-IOV capable NICs.
+    Kubeconfig located in CLUSTER_NAME subfolder.
```

Comment:
Why do you need to save the kubeconfig in a folder?

Reply:
It is the current procedure implemented to gather the kubeconfig file by sharing workspace amongst tasks.

Reply:
So maybe we should mention here `workspace path` or something like this?

Reply:
```suggestion
    CLUSTER_NAME indicates the subfolder where the kubeconfig file from the managed cluster lives in the input workspace.
```

Reply:
```suggestion
CLUSTER_NAME indicates the subfolder where the kubeconfig file from the managed cluster lives in the input workspace.```